### PR TITLE
Update test script

### PR DESCRIPTION
**Pull request by loay**
### Description
The old script `"./run-tests"` was not running.

### CODE REVIEW

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by raymondfeng** [comment]
Is the issue related to Windows? If so, you should create a `.bat` or `.cmd` version of the script.

https://github.com/strongloop/loopback-connector-oracle/blob/master/run-tests customizes the `UV_THREADPOOL_SIZE` env var to allow the connection pooling for Oracle to be large enough for parallel execution of SQL statements.

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

File: test/init/init.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
 // restricted by GSA ADP Schedule Contract with IBM Corp.
 'use strict';
 
+process.env.UV_THREADPOOL_SIZE = 64;
```
Comment (raymondfeng) [inline test/init/init.js]: Is it still needed?

Comment (loay) [inline test/init/init.js]: unfortunately, I couldn't use the `mocha-test` simple script as it failed to run on Windows again for some reason .. I used `mocha.opts` instead .

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
I made few changes such as increasing  `UV_THREADPOOL_SIZE` in order to overcome the timeout.
Also, I couldn't use `test` that runs `mocha-test` as it was working locally but not on the windows server. 
```
Error is : throw new Error("cannot resolve path (or pattern) '" + path + "'");
It works locally but not on the windows remote server
```

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
That failure on nvm 4 is random and not a real failure.

**Pull request merged**